### ISSUE TITLE
Issue with the Lambda Stack Docker build / problems fetching from lambdalabs

### ISSUE THREAD

**Issue by patterntrade** [lambdal/lambda-stack-dockerfiles#4]
The Bionic build>

```
E: Failed to fetch http://archive.lambdalabs.com/ubuntu/pool/main/n/nvidia-cuda-toolkit/libaccinj64-10.0_10.0.130-0lambda3_amd64.deb Undetermined Error [IP: 52.218.160.47 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

The Xenial build>

```
Err:236 http://archive.lambdalabs.com/ubuntu xenial/main amd64 libaccinj64-10.0 amd64 10.0.130-0lambda3~16.04
Hash Sum mismatch
```

Both builds fail because of just one of around a thousand "get"s, and in both cases it’s related to lambdalabs’ archives.

I’ve tried to correct for the Xenial build’s hash sum issue, at times it’s related to the compression algorithm used by apt/get, but haven’t been able to fix it.

It would be great if Lambdalabs or others could try build the images as well, find the root cause, and have it fixed.

As Lambda Stack is such a great (and largely undiscovered) asset, it would be great if this could be remedied.

Kind regards

G

**Comment by sclarkson** [comment]
You're behind a corporate proxy, almost certainly.

See #1 and try the fix in #3

**Comment by patterntrade** [comment]
Hi!

I’m behind a VPN, but not behind something corporate. But probably the VPN has interfered. Anyway, I’ve managed to build the image now.

Thanks so much anyway. :-)


On 30 January 2019 at 20:29:32, sclarkson (notifications@github.com) wrote:

You're behind a corporate proxy, almost certainly.

See #1 and try the fix in #3
